packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -112,6 +114,16 @@ interface BaseConfirmationModalProps {
    */
   readonly variation?: "work" | "destructive";
 
+  /**
+   * Size of the modal (small, large),
+   */
+  readonly size?: "small" | "large";
+
+  /**
+   * Child component. Not displayed if **message** prop is passed.
+   */
+  readonly children?: ReactNode;
```

Comment:
@igor-m-jobber  I tested this in the mdx Playground and I was able to see both, the message and the children I added 🤔 
Possibly because the TS don't validate this file but, I was wondering if we should add an explicit validation for this behavior. Any thoughts?
![Screen Shot 2023-01-12 at 16 18 07](https://user-images.githubusercontent.com/19353565/212161396-f2c62b1d-09b0-4eb1-ace5-996cdffe1d29.png)

Reply:
Ohhh! That is a good one @danlaurent!
You got it right! I was counting on the TS to show you the warning. So this way developer will have to make only 1 choice. But looks like TS doesn't work in the playground.
We definitely should add something. And your ternary operator code below should do the trick!

I will update the code. Thank you!